cartoframes/client/sql_client.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import absolute_import
+
+from carto.exceptions import CartoException
+
+from .internal import create_client
+
+
+class SQLClient(object):
+    """SQLClient class is a client to run SQL queries in a CARTO account.
+
+    Args:
+        creds (:py:class:`Credentials <cartoframes.auth.Credentials>`):
+          A :py:class:`Credentials <cartoframes.auth.Credentials>`
+          instance can be used in place of a `base_url`/`api_key` combination.
+        session (requests.Session, optional): requests session. See `requests
+          documentation
+          <http://docs.python-requests.org/en/master/user/advanced/>`__
+          for more information.
+
+    Example:
+
+        .. code::
+
+            from cartoframes.auth import Credentials
+            from cartoframes.client import SQLClient
+
+            creds = Credentials(username='<YOUR USER NAME>', api_key='<YOUR API KEY>')
+            sql_client = SQLClient(creds)
+
+            sql_client.query('SELECT * FROM table_name')
+            sql_client.execute('DROP TABLE table_name')
+
+            sql_client.distinct('table_name', 'column_name')
+            sql_client.count('table_name')
+            ...
+    """
+
+    def __init__(self, credentials, session=None):
+        self._is_org_user = None
+        self._creds = credentials
+        self._client = create_client(credentials, session)
+
+    def query(self, query, verbose=False):
+        """Run a SQL query. It returns a JSON object with the response.
```

Comment:
JSON -> Python `dict`

Reply:
Actually, it will be a "`list` of `dict`s"?

Reply:
It's a dict

Reply:
I'm getting a list of dictionaries when I run SQLClient(...).query(...):
<img width="480" alt="Screen Shot 2019-07-12 at 8 14 40 AM" src="https://user-images.githubusercontent.com/1041056/61127391-26c7b400-a47d-11e9-81e7-16ed2b3265ac.png">

I think we should use `dict` in place of JSON too since `dict` is the Python version of JSON

Reply:
Well, true. When `verbose=False` it returns the content of 'rows' which is a list. If `verbose=True` it returns a dict with the complete SQL response.

Regarding `JSON`, I thought I changed it, sorry :)

Reply:
Ah, sorry, I . missed the `verbose` option